Makefile

```diff
@@ -175,6 +175,9 @@ upgrade-test-index-image:
 upgrade-test-operator-sdk:
 	./hack/upgrade-test-operator-sdk.sh
 
+consecutive-upgrade-test-operator-sdk:
```

Comment:
nit: not sure we need the `-operator-sdk` suffix. It makes the target name too long. The existing `upgrade-test-operator-sdk` target name is to differentiate it from the `upgrade-test` target.

Reply:
Yes, it's too long, I agree. Does `consecutive-operator-upgrades` make sense?

Reply:
Makes sense. Changed.